src/libtiled/mapreader.cpp

```diff
@@ -1142,6 +1142,9 @@ std::unique_ptr<ImageLayer> MapReaderPrivate::readImageLayer()
     auto imageLayer = std::make_unique<ImageLayer>(name, x, y);
     readLayerAttributes(*imageLayer, atts);
 
+    imageLayer->setRepeatHorizontally(atts.value(QLatin1String("hrepeat")).toInt());
+    imageLayer->setRepeatVertically(atts.value(QLatin1String("vrepeat")).toInt());
```

Comment:
I know other attributes are already not consistent (we have `parallaxx` but `hflip`), I'd still prefer to change this to `repeatx` and `repeaty`.

Reply:
Done.